src/Symfony/Component/HttpKernel/Tests/HttpKernelTest.php

```diff
@@ -112,7 +113,7 @@ public function testHandleHttpException()
     /**
      * @dataProvider getStatusCodes
```

Comment:
If this test is for legacy code it should contain `@group legacy`, changing the name is not fully correct (i.e. in Symfony code base there tests that don't have `Legacy` in name but contain `@group legacy` https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Form/Tests/ChoiceList/Factory/CachingFactoryDecoratorTest.php#L158-L161)